src/main/java/org/alliancegenome/curation_api/services/helpers/validators/AGMDiseaseAnnotationValidator.java

```diff
@@ -38,7 +42,7 @@ public AGMDiseaseAnnotation validateAnnotation(AGMDiseaseAnnotation uiEntity) {
         AffectedGenomicModel subject = validateSubject(uiEntity, dbEntity);
         if(subject != null) dbEntity.setSubject(subject);
 
-        DiseaseRelation relation = validateDiseaseRelation(uiEntity, dbEntity);
```

Comment:
Did the DiseaseRelation enum get cleaned up? Didn't see the delete of that code?